syntax = "proto3";

package chatroom.msgpb;

message CreateGroup {
    uint32 user_id = 1;
    string group_name = 2;
}